Bottom: 97ece09240c2985316f562a7cf690e9cb3850de7
Top:    0e46772ffb23b394460b1fcab3d1fc2c50332bb9
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-07 17:17:32 +0800

Refresh of autofs-5.0.7-probe-each-nfs-version-in-turn-for-singleton-mounts.patch

---

diff --git a/modules/mount_nfs.c b/modules/mount_nfs.c
index c6400f0..81ba3ca 100644
--- a/modules/mount_nfs.c
+++ b/modules/mount_nfs.c
@@ -190,7 +190,7 @@ int mount_mount(struct autofs_point *ap, const char *root, const char *name, int
 	 * to NFSv3 (if it can). If the NFSv4 probe fails then probe as
 	 * normal.
 	 */
-	if (!this->next &&
+	if (!hosts->next &&
 	    mount_default_proto == 4 &&
 	    vers & NFS_VERS_MASK != 0 &&
 	    vers & NFS4_VERS_MASK != 0) {
